packages/core/src/loadPolicy.js

```diff
@@ -11,7 +11,7 @@ async function loadPolicy ({ debugMode, policyPath }) {
     const configSource = fs.readFileSync(policyPath, 'utf8')
     policy = JSON.parse(configSource)
   } else {
-    if (debugMode) console.warn('Lavamoat could not find policy')
+    if (debugMode) console.warn(`Lavamoat could not find policy at ${policyPath}`)
```

Comment:
```suggestion
    if (debugMode) console.warn(`Lavamoat could not find policy at "${policyPath}"`)
```

Reply:
i tend to always wrap vars in logs in quotes, so they dont look weird and empty

Reply:
I just copy-pasted it from 5 lines above :see_no_evil: 
I tend to use single quotes. Will update both.